### DISCUSSION TITLE
IRAF Overscan Parameters Error

### DISCUSSION THREAD

**Discussion by Maciasi** [iraf-community/iraf#251] [Q&A]
Hello!

I am an undergraduate student that is beginning to learn to use IRAF, and unfortunately, I have been stuck on the overscan parameters section with a "cannot open image error." I have been running the software on Ubuntu with VirtualBox, and have been following the reduction process with this website: https://www.subarutelescope.org/Observing/Instruments/HDS/hdsql-e.html#overscan

I have all the data copied and in my iraf directory, so the path should be correct. I am not sure how to fix the problem with the "hedit" and "imcopy" tools, since, from the beginning, I have had issues with setting up the IRAF software.

Please let me know if you have any thoughts, thank you!

<img width="466" alt="overscan error" src="https://user-images.githubusercontent.com/103520447/209725518-d5716253-19dc-4118-bb05-c9f28a42b390.PNG">

**Comment by olebole** [comment]
My guess would be that the path to the file is incorrect; specifically a leading '**/**' may be missing to indicate it is an absolute path. If this is not the case, it is rather a problem with the hds software and not with IRAF itself; you should look there for support.

However, I would also like to ask you to consider the following: IRAF is really old software, nowadays lacks further development and and institutional support. It is mainly maintained for those who cannot switch yet. Unless there is a real strong requirement, it is a really bad idea to start with it for a new project. It would be much better to use modern Python projects like [Astropy](https://astropy.org) instead whenever it is suitable. This is much more future-proof than learning an outdated, 40 year old legacy software system.
